src/token/raft.ts

```diff
@@ -91,6 +104,77 @@ export class RaftToken {
     return newVeRaft.div(newTotalVeRaft).mul(annualGiveAway).div(stakeAmount);
   }
 
+  public async getBalancerPoolData(): Promise<SubgraphPoolBase | null> {
```

Comment:
Is this function used outside of the SDK? If not, let's make it private.

Reply:
if price impact needs to be calculated multiple times, then yes. queried pool data and pass it as param to `calculatePriceImpact()`. it's by design to keep it public.
however at this moment we only call it once on app side. i'd suggest we still keep it as public right now.